src/testing/saml-idp/Dockerfile.build

```diff
@@ -11,7 +11,7 @@
 # image (Dockerfile.idp) because we need to mount ../vendor into the container
 # as part of the build process, and there's no way to mount an external
 # directory into a docker container as part of 'docker build'
-FROM golang:1.10.2-alpine AS build
+FROM golang:1.16.4-alpine AS build
```

Comment:
1.10 seems almost intentionally old.  Are we trying to ensure backwards compatibility with this test?

Reply:
@actgardner Is this intentional?

Reply:
@actgardner Can we delete the whole test/saml-idp directory?

Reply:
I believe so, yes.